Cargo.toml

```diff
@@ -6,18 +6,14 @@ edition = "2021"
 
 [dependencies]
 async-broadcast = "0.7"
-async-compatibility-layer = { version = "1.1", default-features = false, features = [
-    "logging-utils",
-] }
 async-lock = "2.8"
```

Comment:
can we also update this async--lock to "3" here so that we can compile sequencer without lockfile

Reply:
Sure